api-reference/10 UI Components/GridBase/1 Configuration/filterRow/showOperationChooser.md

```diff
@@ -8,3 +8,4 @@ default: true
 Specifies whether icons that open the filter lists are visible.
 
 ---
+[note] If certain columns do not display the filter icons, specify the columns' [dataType]({basewidgetpath}/Configuration/columns/#dataType) and make sure that you did not disable their [allowFiltering]({basewidgetpath}/Configuration/columns/#allowFiltering) property.
```

Comment:
```suggestion
[note] If columns do not display filter icons, specify the columns' [dataType]({basewidgetpath}/Configuration/columns/#dataType) and ensure that the columns' [allowFiltering]({basewidgetpath}/Configuration/columns/#allowFiltering) property is not disabled. This will allow filter icons to be  displayed.
```